<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>841</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>139</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="txt_user_id">
      <property name="placeholderText">
       <string>ID del usuario</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_name">
      <property name="placeholderText">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_email">
      <property name="placeholderText">
       <string>Correo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmb_role">
      <item>
       <property name="text">
        <string>Profesor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Estudiante</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>395</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_add_user">
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_update_user">
      <property name="text">
       <string>Actualizar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_delete_user">
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
